test/repl_test.js

```diff
@@ -13,6 +13,8 @@ var colors = require('colors')
 var payload = ''
 var count = 0
 
+var PADDING = 19
```

Comment:
this is basically how gulp injects includes and links in a html file, with the help of placeholders :stuck_out_tongue: